lib/rubygems/commands/push_command.rb

```diff
@@ -107,4 +120,15 @@ def get_hosts_for(name)
   def get_push_scope
     :push_rubygem
   end
+
+  def get_attestations_part
+    bundles = "[" + options[:attestations].map do |attestation|
+      Gem.read_binary(attestation)
+    end.join(",") + "]"
+    [
+      "attestations",
+      bundles,
+      { content_type: "application/json" },
+    ]
```

Comment:
Please forgive my nitpicking: how about this?

```suggestion
    bundles = options[:attestations].map do |attestation|
      Gem.read_binary(attestation)
    end.join(",")
    [
      "attestations",
      "[#{bundles}]",
      { content_type: "application/json" },
    ]
```